api/resources_portal/management/commands/send_weekly_digest_emails.py

```diff
@@ -0,0 +1,103 @@
+import datetime
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from resources_portal import utils
+from resources_portal.emailer import (
+    EMAIL_SOURCE,
+    LOGO_EMBEDDED_IMAGE_CONFIGS,
+    PLAIN_TEXT_EMAIL_FOOTER,
+    send_mail,
+)
+from resources_portal.models import User
+
+EMAIL_HTML_TEMPLATE = (
+    Path("resources_portal/email_assets/weekly_email_template_inlined.html")
+    .read_text()
+    .replace("\n", "")
+)
+EMAIL_HTML_LIST_ITEM = (
+    Path("resources_portal/email_assets/replace_notification_list_inlined.html")
+    .read_text()
+    .replace("\n", "")
+)
+CTA_HTML = (
+    Path("resources_portal/email_assets/cta_templated_inlined.html").read_text().replace("\n", "")
+)
+
+
+def get_plain_text_for_notification(notification):
+    email_dict = notification.get_email_dict()
+    plain_text_email_body = email_dict["plain_text_email_body"]
+
+    return f"{notification.human_readable_date}\n{plain_text_email_body}\n"
+
+
+def get_formatted_text_for_notification(notification):
+    email_dict = notification.get_email_dict()
+    pretty_time = utils.pretty_date(notification.created_at)
```

Comment:
Have you checked that this outputs times in the user's timezone and not UTC? Django claims to have timezone support, which stores times in UTC and translates to the user's timezone in forms and templates. This might already be working automagically, but I would check. Reference: https://docs.djangoproject.com/en/3.1/topics/i18n/timezones/

Reply:
So this actually conveniently sidesteps that issue because `pretty_time` returns things like "5 minutes ago" or "3 days ago". Since both the created_at and the `timezone.now()` function use django times, it should be consistent. Additionally when we say "For the week of" we only specify down to the day, so that should be timezone safe as well.

This is actually really really convenient because we don't store what timezone each user is in.

Reply:
Ah delightful. Good to go!